sdks/configure

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh -e
+
+function show_help () {
+cat <<EOL
+Usage: configure [options]
+    -h, --help       Displays this help
+    --ios            Enable IOS build (disabled by default)
```

Comment:
Should we use `--enable-{ios,android,wasm}` to be consistent with `--disable-bcl`?